cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/PushCommand.java

```diff
@@ -204,7 +222,16 @@ public void setParentCmdParser(CommandLine parentCmdParser) {
     private void pushPackage(BuildProject project) {
         Path balaFilePath = validateBalaFile(project);
         pushBalaToCustomRepo(balaFilePath);
-        outStream.println("Successfully pushed " + userDir.relativize(balaFilePath)
+
+        Path relativePathToBalaFile;
+
+        try {
+            relativePathToBalaFile = project.sourceRoot().relativize(balaFilePath);
+        } catch (IllegalArgumentException e) {
```

Comment:
Why do we need to catch `IllegalArgumentException` here?

Reply:
When using relative paths for custom target directories, an IllegalArgumentException is thrown. In those cases we catch the exception and handle that case by simply printing the actual custom path.

Reply:
We do not need to relativize all the time. If the user has given the path, then we should print it as it is. Else, if is the default one then we print the relative path which is already being done ATM.

Reply:
Changed the implementation based on if the custom path is given or not